WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -146,7 +147,14 @@ class GutenbergViewController: UIViewController, PostEditor {
 
     // MARK: - Private variables
 
-    private lazy var gutenberg = Gutenberg(dataSource: self)
+    private lazy var gutenbergImageLoader: GutenbergImageLoader = {
+        return GutenbergImageLoader(post: self.post)
+    }()
+
+    private lazy var gutenberg: Gutenberg = {
+        return Gutenberg(dataSource: self, extraModules: [self.gutenbergImageLoader])
```

Comment:
```suggestion
        return Gutenberg(dataSource: self, extraModules: [gutenbergImageLoader])
```

No need for `self.` in this initializer